src/gsi/setupbend.f90

```diff
@@ -111,6 +111,7 @@ subroutine setupbend(obsLL,odiagLL, &
 !   2024-12-04  Li       - remove the QC check for rejecting MetOp data <8 km
 !   2024-12-04  Li       - add GRACE-FO (803&804) data 
 !   2024-12-04  Li       - add new obs error model by Chris Riedel
+!   2025-12-08  Li       - add PlanetiQ YAM-8 (768) data
```

Comment:
We don't really need to add comments like this anymore since GSI source code is version controlled.  git maintains a record of the change history for each file. 

You don't need to remove this comment but moving forward I don't think you need to add comments like this.